src/client/src/pages/LoginPage.jsx

```diff
@@ -74,38 +71,31 @@ const BACKEND_URI = process.env.REACT_APP_FRONTEND_URI || 'http://localhost:8080
           uid: userCredential.user.uid,
           email: userCredential.user.email,
           accessToken: userCredential.user.stsTokenManager.accessToken,
-          refreshToken: userCredential.user.stsTokenManager.refreshToken,
-          expirationTime: userCredential.user.stsTokenManager.expirationTime,
+          // refreshToken: userCredential.user.stsTokenManager.refreshToken,
+          // expirationTime: userCredential.user.stsTokenManager.expirationTime,
         };
         dispatch(init(obj));
+
         const expires = new Date();
         expires.setTime(obj.expirationTime);
         setCookie('access_token', obj.accessToken, {
           path: '/',
           expires,
         });
-        setCookie('refresh_token', obj.refreshToken, {
-          path: '/',
-          expires,
-        });
-        setCookie('expiration_time', obj.expirationTime, {
+        setCookie('uid', obj.uid, {
           path: '/',
           expires,
         });
+        // setCookie('refresh_token', obj.refreshToken, {
+        //   path: '/',
+        //   expires,
+        // });
+        // setCookie('expiration_time', obj.expirationTime, {
+        //   path: '/',
+        //   expires,
+        // });
         navigate('/home');
       })
-      .then ((userCredential) => {
```

Comment:
why do we remove this?

Reply:
won't we need to refer back to the user data? or this already handled by the cookie?

Reply:
It's in the cookies! This was only setting it for the current page which doesn't do anything for us, if you want to keep it persistent you have to either update the store or the cookies, both of which are already doje

Reply:
sweet, lgtm